soul-common/src/main/java/org/dromara/soul/common/utils/UpstreamCheckUtils.java

```diff
@@ -47,17 +47,18 @@ public static boolean checkUrl(final String url) {
         if (StringUtils.isBlank(url)) {
             return false;
         }
-        if (checkIP(url)) {
-            String[] hostPort;
-            if (url.startsWith(HTTP)) {
-                final String[] http = StringUtils.split(url, "\\/\\/");
-                hostPort = StringUtils.split(http[1], Constants.COLONS);
-            } else {
-                hostPort = StringUtils.split(url, Constants.COLONS);
-            }
-            return isHostConnector(hostPort[0], Integer.parseInt(hostPort[1]));
+        String[] hostPort;
+        if (url.startsWith(HTTP)) {
+            final String[] http = StringUtils.split(url, "\\/\\/");
+            hostPort = StringUtils.split(http[1], Constants.COLONS);
         } else {
-            return isHostReachable(url);
+            hostPort = StringUtils.split(url, Constants.COLONS);
+        }
+        final int port = hostPort.length > 1 ? Integer.parseInt(hostPort[1]) : 80;
```

Comment:
It it possible to support ` https(443)`?

Reply:
done